### PR TITLE
Fixed the epoch activation in transactions pre-processors

### PR DESCRIPTION
**Pull request by iulianpascalau**
- fixed the epoch activation in transactions pre-processors

### CODE REVIEW

File: process/block/shardblock.go [hunk 1]
```diff
@@ -328,6 +329,11 @@ func (sp *shardProcessor) ProcessBlock(
 		return err
 	}
 
+	if header.GetNonce() == 56430 {
```
Comment (sasurobert) [inline process/block/shardblock.go:332]: Delete this

Comment (iulianpascalau) [reply process/block/shardblock.go:332]: Of course. Will do!

Comment (iulianpascalau) [reply process/block/shardblock.go]: done

**Review by sasurobert — APPROVED** [review]

**Review by AdoAdoAdo — APPROVED** [review]

**Review by gabi-vuls — APPROVED** [review]
System test passed

**Pull request merged**